loadtest/store/memstore/store.go

```diff
@@ -193,6 +195,21 @@ func (s *MemStore) SetEmojis(emoji []*model.Emoji) error {
 	return nil
 }
 
+func (s *MemStore) SetReaction(reaction *model.Reaction) error {
+	if s.reactions[reaction.PostId] == nil {
+		s.reactions[reaction.PostId] = map[string]map[string]*model.Reaction{}
+		s.reactions[reaction.PostId][reaction.EmojiName] = map[string]*model.Reaction{}
+	} else if s.reactions[reaction.PostId][reaction.EmojiName] == nil {
+		s.reactions[reaction.PostId][reaction.EmojiName] = map[string]*model.Reaction{}
+	}
+	s.reactions[reaction.PostId][reaction.EmojiName][reaction.UserId] = reaction
```

Comment:
I've puto `emojiName` first because I think the logic path to check for reactions is first by post and then my emoji